ch_pipeline/core/dataquery.py

```diff
@@ -72,12 +72,16 @@ class QueryDatabase(task.MPILoggedTask):
         host and directory in which to find data.
     start_time, end_time : string (default: None)
         start and end times to restrict the database search to
-        can be in any format ensure_unix will support, including eg
+        can be in any format ensure_unix will support, including e.g.
         20190116T150323 and 2019-1-16 08:03:23 -7
-    start_csd, end_csd : float
-        Start and end CSDs. Only used if `start_time` is not set.
-    instrument : string (default: 'chimestack')
-        data set to use
+    acqtype : string (default: 'corr')
+        Type of acquisition. Options for acqtype are: 'corr', 'hk', 'weather',
+        'rawadc', 'gain', 'flaginput', 'digitalgain'.
```

Comment:
`gain` is not supported in `LoadDataFiles`, are dataquery, and `LoadDataFiles` related?

Reply:
Good point. I've matched the LoadDataFiles names up to the acqtype names.